### PR TITLE
add medium severity to tests

### PR DESCRIPTION
**Pull request by jeffreygolden**
https://app.shortcut.com/replicated/story/37175/ci-should-fail-on-image-cves-that-do-not-meet-our-policy

### CODE REVIEW

**Review by DanStough — APPROVED** [review]

**Comment by emosbaugh** [comment]
Looks like our policy is as follows:

Critical | Within 2 weeks
High | Within 60 days
Medium | Within 90 days

**Review by emosbaugh — APPROVED** [review]

**Comment by jeffreygolden** [comment]
image

File: .github/workflows/kotsadm.yaml [hunk 1]
```diff
@@ -527,137 +527,153 @@ jobs:
...
+          template: '@/contrib/sarif.tpl'
+          output: 'kurl-scan-output.sarif'
+          exit-code: '0'
+          ignore-unfixed: true
+          severity: 'CRITICAL,HIGH,MEDIUM'
       - name: Upload scan report
         uses: github/codeql-action/upload-sarif@v1
         with:
-          sarif_file: ${{ steps.scan.outputs.sarif }}
+          sarif_file: kurl-scan-output.sarif
```
Comment (emosbaugh) [inline .github/workflows/kotsadm.yaml:632]: kurl-proxy-scan-output.sarif

File: .github/workflows/kotsadm.yaml [hunk 2]
```diff
@@ -527,137 +527,153 @@ jobs:
...
+          template: '@/contrib/sarif.tpl'
+          output: 'migration-scan-output.sarif'
+          exit-code: '0'
+          ignore-unfixed: true
+          severity: 'CRITICAL,HIGH,MEDIUM'
       - name: Upload scan report
         uses: github/codeql-action/upload-sarif@v1
         with:
-          sarif_file: ${{ steps.scan.outputs.sarif }}
+          sarif_file: migration-scan-output.sarif
```
Comment (emosbaugh) [inline .github/workflows/kotsadm.yaml:588]: kotsadm-migration-scan-output.sarif

File: .github/workflows/kotsadm.yaml [hunk 3]
```diff
@@ -527,137 +527,153 @@ jobs:
...
+          template: '@/contrib/sarif.tpl'
+          output: 'postgres-stretch-scan-output.sarif'
+          exit-code: '0'
+          ignore-unfixed: true
+          severity: 'CRITICAL,HIGH,MEDIUM'
       - name: Upload scan report
         uses: github/codeql-action/upload-sarif@v1
         with:
-          sarif_file: ${{ steps.scan.outputs.sarif }}
+          sarif_file: postgres-stretch-scan-output.sarif
```
Comment (emosbaugh) [inline .github/workflows/kotsadm.yaml:566]: postgres-scan-output.sarif

File: .github/workflows/kotsadm.yaml [hunk 4]
```diff
@@ -527,137 +527,153 @@ jobs:
...
+          template: '@/contrib/sarif.tpl'
+          output: 'postgres-scan-output.sarif'
+          exit-code: '0'
+          ignore-unfixed: true
+          severity: 'CRITICAL,HIGH,MEDIUM'
       - name: Upload scan report
         uses: github/codeql-action/upload-sarif@v1
         with:
-          sarif_file: ${{ steps.scan.outputs.sarif }}
+          sarif_file: postgres-scan-output.sarif
```
Comment (emosbaugh) [inline .github/workflows/kotsadm.yaml:544]: postgres-alpine-scan-output.sarif

**Review by emosbaugh — APPROVED** [review]
can you make the sarif file match the image name?

**Pull request merged**